src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Commands.ScenarioTests.ResourceManager.Common.csproj

```diff
@@ -12,10 +12,7 @@
     <RootNamespace>Microsoft.Azure.Commands.ScenarioTest</RootNamespace>
```

Comment:
Looks like there are some unintended changes. Can you please revert changes to these files?